tools/ci_connector_ops/ci_connector_ops/qa_engine/models.py

```diff
@@ -5,23 +5,28 @@
 
 from enum import Enum
 from typing import List
+
 from pydantic import BaseModel
 
 class ConnectorTypeEnum(str, Enum):
     source = "source"
     destination = "destination"
 
 class ReleaseStageEnum(str, Enum):
+    unknown = "unknown"
```

Comment:
What unknown connectors did you find?

Reply:
* source-freshcaller
* source-genesys
* source-rss
* destination-e2e-testing